sources/Outputs/src/Diagram.cpp

```diff
@@ -70,14 +70,15 @@ Diagram::writeTable(const algo::GeneratorDefinition& generator, std::stringstrea
   buffer << numberLines;
   buffer << ",2)";
 
+  int divisorFactor = 100;
```

Comment:
Please add a const identifier and a static

Reply:
> Please add a const identifier and a static

const, yes, static not necessary here the function is called ony once

Reply:
OK then, const will be enough :)